MapboxNavigation/FeedbackViewController.swift

```diff
@@ -165,12 +205,42 @@ class FeedbackViewController: UIViewController, DismissDraggable, UIGestureRecog
         progressBar.trailingAnchor.constraint(equalTo: view.trailingAnchor).isActive = true
         progressBar.bottomAnchor.constraint(equalTo: view.safeBottomAnchor).isActive = true
     }
+    
+    func defaultSendFeedbackHandler(source: FeedbackSource = .user, uuid: UUID) -> (FeedbackItem) -> Void {
```

Comment:
There’s only one call site and it doesn’t specify the `source` argument, so inline the default `.user` value below.